src/main/resources/application.yml

```diff
@@ -77,4 +87,13 @@ management:
       cache:
         time-to-live: 2000ms
 
+
+services:
+  prisons-api:
```

Comment:
Can we call this `prison-api` to match the API name - this is different from the approved premises project's naming, but I think they may have got it wrong...